### PR TITLE
Make sure "Flask App" docs include all app attributes

### PR DESCRIPTION
**Pull request by djmitche**
https://api.pub.build.mozilla.org/docs/development/@relengapi/app/ probably doesn't list everything it should.

### CODE REVIEW

File: docs/development/@relengapi/app.rst [hunk 1]
```diff
@@ -5,20 +5,47 @@ The RelengAPI Flask App is mostly a normal Flask App, but has a few additional a
...
+
+        This attribute gives a list of all RelengAPI blueprints.
+        It is different from ``current_app.blueprints`` in that it does not include blueprints from Flask extensions.
+        Each RelengAPI blueprint has a ``dist`` attribute giving the SetupTools distribution from which the blueprint came.
 
-    .. py:attribute:: authz
 
-        See :doc:`auth`
+Blueprints can add their own attributes to the application as necessary.
+These should generally begin with the name of the blueprint to avoid name collisions.
```
Comment (Callek) [inline docs/development/@relengapi/app.rst]: would be great if we could come up with a way to have apps hook docs into here (with something like ```[from relengapi-mapper]``` annotation) I don't think thats easy at present so not blocking

File: docs/development/@relengapi/app.rst [hunk 2]
```diff
@@ -5,20 +5,47 @@ The RelengAPI Flask App is mostly a normal Flask App, but has a few additional a
...
+
+    .. py:attribute:: memcached
+
+        Access to configured memcached caches.
+        See :doc:`memcached`
+
+    .. py:attribute:: relengapi_blueprionts
+
+        Access to configured memcached caches.
+        See :doc:`memcached`
```
Comment (Callek) [inline docs/development/@relengapi/app.rst]: duplicated attribute "relengapi_blueprints" (with typo) but listing docs for memcached. I suspect this is either "needs to be deleted" or you meant to list something else here.

**Comment by Callek** [comment]
:+1: once comments addressed

File: docs/development/@relengapi/app.rst (hunk 1 above)
Comment (djmitche) [inline docs/development/@relengapi/app.rst]: I'm not sure that's possible at all with sphinx, tbh..

**Pull request merged**